README.md

```diff
@@ -36,6 +36,27 @@ It will install WireGuard (kernel module and tools) on the server, configure it,
 
 Run the script again to add or remove clients!
 
+## Configure clients
+
+### RHEL 8
+
+Install the necessary packages:
+
+```
+sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+sudo dnf install dnf-plugins-core
+sudo dnf config-manager --enable elrepo
+sudo dnf install kmod-wireguard wireguard-tools
+```
+
+Create file `/etc/wireguard/wg0.conf` with contents of the client config generated by the script (copy paste from server).
+
```

Comment:
Can also just cp /mv wg0-client-foo.conf to /etc/wireguard/wg0.conf as well.  I recommend cp as you can use the original as a safety or template to hack on for tweaks/troubleshooting.